documentation/docs/13-cli.md

```diff
@@ -2,43 +2,7 @@
 title: Command Line Interface
 ---
 
-SvelteKit includes a command line interface for building and running your app.
-
-In the default project template `svelte-kit dev`, `svelte-kit build` and `svelte-kit preview` are aliased as `npm run dev`, `npm run build` and `npm run preview` respectively. You can also invoke the CLI with [npx](https://www.npmjs.com/package/npx):
-
-```bash
-npx svelte-kit dev
-```
-
-### svelte-kit dev
-
-Starts a development server. It accepts the following options:
-
-- `-p`/`--port` — which port to start the server on
-- `-o`/`--open` — open a browser tab once the server starts
-- `--host` — expose the server to the network.
-- `--https` — launch an HTTPS server using a self-signed certificate. Useful for testing HTTPS-only features on an external device
-
-> This command will fail if the specified (or default) port is unavailable. To use an alternative port instead of failing, set the [`config.kit.vite.server.strictPort`](/docs/configuration#vite) option to `false`.
-
-### svelte-kit build
-
-Builds a production version of your app, and runs your adapter if you have one specified in your [config](/docs/configuration). It accepts the following option:
-
-- `--verbose` — log more detail
-
-After building the app, you can reference the documentation of your chosen [adapter](/docs/adapters) and hosting platform for specific instructions on how to serve your app.
-
-### svelte-kit preview
-
-After you've built your app with `svelte-kit build`, you can start the production version (irrespective of any adapter that has been applied) locally with `svelte-kit preview`. This is intended for testing the production build locally, **not for serving your app**, for which you should always use an [adapter](/docs/adapters).
-
-Like `svelte-kit dev`, it accepts the following options:
-
-- `-p`/`--port`
-- `-o`/`--open`
-- `--host`
-- `--https`
+SvelteKit is built with Vite and so you will mostly use its CLI. However, SvelteKit includes a command line interface for packaging libraries.
```

Comment:
note to self: without `dev`/`build`/`preview`, this paragraph transitions to the next quite strangely